makefile

```diff
@@ -0,0 +1,36 @@
+## Manage python dependencies:
```

Comment:
Traditionally Makefiles are named with capital M. Not sure why.

Reply:
Will change